drv/transceivers-server/src/udp.rs

```diff
@@ -168,6 +100,81 @@ impl ServerImpl {
         }
     }
 
+    fn handle_packet(
+        &mut self,
+        mut meta: UdpMetadata,
+        rx_data_buf: &[u8],
+        tx_data_buf: &mut [u8],
+    ) {
+        // Modify meta.size based on the output packet size
```

Comment:
I think this comment should move down to line 119 or 120?

Reply:
Yes, done in 2f7c6bf9b6de413d63fcaaa7005d3a7b78319c4a